### ISSUE TITLE
Button skin

### ISSUE THREAD

**Issue by robeson1010** [Nixxen/TrinketMenu#2]
Is there any way that the trinket button can work with masque.
https://github.com/StormFX/Masque

**Comment by Nixxen** [comment]
Honestly I just picked this addon up since it wasn't working for Classic and I wanted to use it.
I'm still learning and don't know much at all about coding as it's mostly just a hobby I dabble with for fun.

I looked at the Masque API wiki (https://github.com/StormFX/Masque/wiki/Masque-API), and it looks like it could be used to skin the buttons, but it would require a rewrite of how the buttons are displayed in game.

So in short, yes, I strongly suspect it is possible - But I doubt I am the one that's able to add it at this point in time ;)

**Comment by TigerVI** [comment]
Since I'm also new at WOW Lua and Masque API I tried to add it for myself and it seems only a few lines need to be added:
Image of the result (using ElvUIesque Skin): https://imgur.com/a/Imj2pci

I  added the following lines at the beginng of Trinketmenu.lua to create the MasqueGroup:
```
local Masque = LibStub("Masque", true)
local MasqueGroup = Masque:Group("TrinketMenu")
```

and the following lines at the end of the function TrinketMenu.Initialize() to add the buttons to the previously defined group.

```
MasqueGroup:AddButton(TrinketMenu_Trinket0)
MasqueGroup:AddButton(TrinketMenu_Trinket1)
for i = 1, 30 do
   _G["TrinketMenu_Menu"..i]:SetFrameLevel(2)
   MasqueGroup:AddButton(_G["TrinketMenu_Menu"..i])
end
```

**Comment by Nixxen** [comment]
Nice job, this is great stuff!
I'll test this during the weekend and see how it runs then push a patch if it's not causing any issues.

**Comment by Nixxen** [comment]
Gave it a quick test.
It works well with Masque installed, but without masque it throws errors (referencing masque, which doesn't exist).
Will need to find a way to only load it if Masque is present.
I'll look into it and see what I can find.

**Comment by TigerVI** [comment]
Using "if masque" around the calls should do the trick. 
Tested it with disabled masque addon on my pc.

```
if (Masque) then	
	local MasqueGroup = Masque:Group("TrinketMenu")	
end	
```

```
if (Masque) then
     MasqueGroup:AddButton(TrinketMenu_Trinket0)
     MasqueGroup:AddButton(TrinketMenu_Trinket1)
end
	
for i = 1, 30 do
    _G["TrinketMenu_Menu"..i]:SetFrameLevel(2)
    if (Masque) then
        MasqueGroup:AddButton(_G["TrinketMenu_Menu"..i])
    end
end
```

**Comment by Nixxen** [comment]
Wasn't as bad as expected TBH.
Some checks pointed at the lib solved it.
I'll get it pushed ASAP.

**Comment by Nixxen** [comment]
Should be active in the latest version.
Let me know if it doesn't work :)

**Comment by TigerVI** [comment]
Just downloaded it, it's working.  
With and without masque installed no lua error messages appear.

Many thanks for your efforts.
